src/sentry/profiles/task.py

```diff
@@ -43,6 +48,18 @@ def process_profile(profile: MutableMapping[str, Any], **kwargs: Any) -> None:
         json.dumps(profile),
     )
 
+    track_outcome(
+        org_id=project.organization_id,
+        project_id=project.id,
+        key_id=None,
```

Comment:
I'm realizing this just now, but we should probably produce the `key_id` for profiles in Relay so we can pipe it through the task and add it here, just like we do for events.